### PR TITLE
Adding Tampermonkey-for-Edge.md to support #1046

### PR DESCRIPTION
**Pull request by JoMangee**
A new wiki page detailing the steps to install Tampermonkey with the Microsoft Edge browser - see #1046

### CODE REVIEW

File: views/includes/documents/Tampermonkey-for-Edge.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+[tampermonkeyEdgeScreenshot2]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge3.png "Edge Extension prompt for Tampermonkey"
+[tampermonkeyEdgeScreenshot3]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge4.png "Tampermonkey installed in Edge"
+[tampermonkeyEdgeScreenshot4]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge5.png "Tampermonkey options in Edge"
+[oujsScriptPageScreenshot1]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/openuserjs_script.gif "Ready to install a script"
+[tampermonkeyEdgeScreenshot4]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge7.png "Installing a script in Edge"
+[tampermonkeyEdgeScreenshot5]: https://raw.githubusercontent.com/wiki/OpenUserJS/OpenUserJS.org/images/tampermonkey_edge5.png "Tampermonkey Dashboard"
+[tampermonkeyForAndroid]: Tampermonkey-for-Android
+[tampermonkeyForChrome]: Tampermonkey-for-Chrome
+[tampermonkeyNet]: http://tampermonkey.net/
```
Comment (Martii) [inline views/includes/documents/Tampermonkey-for-Edge.md]: Needs a final newline here at the end otherwise looks good to go.

Comment (Martii) [inline views/includes/documents/Tampermonkey-for-Edge.md]: Needs a final newline here at the end otherwise looks good to go.

Comment (JoMangee) [inline views/includes/documents/Tampermonkey-for-Edge.md]: Thanks

Comment (Martii) [inline views/includes/documents/Tampermonkey-for-Edge.md]: I'll go ahead and get that... need to remove a few things across the docs. Thanks for your cool addition.

Comment (Martii) [inline views/includes/documents/Tampermonkey-for-Edge.md]: Correction... looks like GH is hiding the newline here. Bleh.

**Pull request merged**